daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -308,3 +308,10 @@ For more information on tracing and logs see the [observability]({{< ref observa
 * [Service invocation overview]({{< ref service-invocation-overview.md >}})
 * [Service invocation API specification]({{< ref service_invocation_api.md >}})
 * [gRPC proxying community call video](https://youtu.be/B_vkXqptpXY?t=70)
+
+## Community call demo
+Watch this [video](https://youtu.be/B_vkXqptpXY?t=69) on how to use Dapr's gRPC proxying capability:
```

Comment:
Do we need the link if the video is embedded?

Reply:
Trying to keep consistent with the other examples we have in the docs.